subscription_platform/explores/daily_active_logical_subscriptions.explore.lkml

```diff
@@ -25,6 +25,14 @@ explore: daily_active_logical_subscriptions {
     ]
   }
 
+  join: subscription_services {
+    from: daily_active_logical_subscriptions__subscription__services
+    sql_table_name: UNNEST(daily_active_logical_subscriptions.subscription.services) ;;
```

Comment:
Ideally this would use a view field reference like `${daily_active_logical_subscriptions.subscription__services}`, but there appears to be a Looker bug where it incorrectly resolves that reference into `daily_active_logical_subscriptions.subscription__services` (and there is no `subscription__services` column, it should be be `subscription.services`).